Initial file installation docs update (#159)
cylc/cylc-doc: src/suite-design-guide/general-principles.rst

Installing Files At Start-up
^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Files can be installed on the remote platform. As standard, Cylc installs the

Reviewer: ```suggestion
Files can be installed on remote platforms. As standard, Cylc installs the
```

```suggestion
Files can be installed on a remote platform. As standard, Cylc installs the
```

following directories: ``app``, ``bin``, ``etc``, ``lib``.
Cylc supports adding custom directories and files to the file installation.
E.g. dir1, dir2, file1, file2, you can add the following configuration to your

> :cylc:conf:`flow.cylc`. To denote a directory, please add a trailing slash.  

Reviewer 2: Could you squeeze in a reference to :cylc:conf:`flow.cylc[scheduler]install` so that people have a nice easy link into the relevant bit of the reference docs.

Installing Files At Start-up
^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Files can be installed on any remote platforms. As standard, Cylc installs the

Reviewer: Two possible changes here. The plural is necessary, the change of "As standard,", to "By default" is my preference only.
```suggestion
Files can be installed on any remote platform. By default Cylc installs the
```

following directories: ``app``, ``bin``, ``etc``, ``lib``.
Cylc supports adding custom directories and files to the file installation.

> E.g. dir1, dir2, file1, file2, you can add the following configuration to your

Reviewer: ```suggestion
If, for example, you wished to install directiories ``dir1``, `dir2``, and files ``file1`` and ``file2``
add the following configuration to
```

:cylc:conf:`flow.cylc`, under the section
:cylc:conf:flow.cylc[scheduler]install.

> To denote a directory, please add a trailing slash.  

Reviewer: ```suggestion
To mark an item as a directory add a trailing slash.
```

.. code-block:: cylc

    [scheduler]
        install = dir1/, dir2/, file1, file2

.. note::

>    Please ensure files and directories to be installed are located in the top

Reviewer: Some smart alec is bound ask why...

Also, I'm not a fan of the "pleases". Perhaps I'm just rude.
